indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -101,18 +105,38 @@ def _process(self):
         return jsonify_data(box_html=tpl.render_decision_box(self.abstract, form, management=self.management))
 
 
-class RHResetAbstractJudgment(RHAbstractReviewBase):
+class RHResetAbstractState(RHAbstractReviewBase):
+    def _checkProtection(self):
+        if (not self.abstract.can_judge(session.user, check_state=True) and
```

Comment:
I'd split this access check into two ifs so you don't have to read it 3 times to be sure what it's supposed to check :)
Or at least explain in a comment when resetting the state should be possible.